runtime/oti/j9nonbuilder.h

```diff
@@ -5207,7 +5207,10 @@ typedef struct J9InternalVMFunctions {
 	I_32 (*invoke31BitJNI_OnXLoad)(struct J9JavaVM *vm, void *handle, jboolean isOnLoad, void *reserved);
 #endif /* defined(J9VM_ZOS_3164_INTEROPERABILITY) && (JAVA_SPEC_VERSION >= 17) */
 #if defined(J9VM_OPT_JFR)
+	jint (*initializeJFR)(struct J9JavaVM *vm, BOOLEAN lateInit);
 	void (*jfrExecutionSample)(struct J9VMThread *currentThread, struct J9VMThread *sampleThread);
+	jboolean (*setJFRRecordingFileName)(struct J9JavaVM *vm, char *fileName);
```

Comment:
Why is the type of the second parameter not `const char *`?

Reply:
`fileName` is allocated via `copyStringToUTF8WithMemAlloc()`, and will be freed if multiple `setJFRRecordingFileName()` calls are made. `j9mem_free_memory` doesn't take `const` pointers.